src/ctia/task/migration/store.clj

```diff
@@ -25,7 +27,19 @@
 
 (def timeout (* 5 60000))
 (def es-max-retry 3)
-(def migration-es-conn (atom nil))
+(defonce migration-es-conn (atom nil))
+
+(s/defschema MigrationStoreServices
+  {:ConfigService {:get-in-config (s/=>* s/Any
+                                         [(s/named s/Any 'path)]
+                                         [(s/named s/Any 'path)
+                                          (s/named s/Any 'default)])}
+   :StoreService {:deref-stores (s/=> s/Any)}})
```

Comment:
> I would hide the atom detail here, and just use something like get-store, wdyt?

Yes, I had the same thought. Since you're interested, I'll see how feasible it is. I was worried that enough places used `stores` with deref/reset!/swap! for it to be too much work to abstract over, but I never checked.

> FYI I am currently removing the use of stores atom in this PR: #968. It's initialization triggers template and mapping updates in the source, which is unnecessary (and not desirable) during the migration.

Ah ok. I'll keep that in mind, might be able to avoid the merge conflict.

Reply:
It was just a matter of searching for `stores-atom` in the big PR. `reset!` is only used to initialize the stores. Should be easy to refactor.